### ISSUE TITLE
Error in Step 3.4 "Build verification test suite"

### ISSUE THREAD

**Issue by jeremydzx** [marg-tools/CoMeT#3]
Sorry to disturb. I'm following your steps to learn CoMeT features. But in step 3.4 "Build verification test suite", all the cases are failed, and one of those error messages is

`[SNIPER] Warning: Unable to use physical addresses for shared memory simulation.
[SNIPER] Running ['/home/dzx/CoMeT/record-trace', '-o', '/tmp/tmp8Pk2EE/run_benchmarks', '-v', '-e', '1', '-s', '0', '-r', '1', '--follow', '--routine-tracing', '--', '/home/dzx/CoMeT/test/test-installation/test']
[SNIPER] Start
[SNIPER] Running ['bash', '-c', '/home/dzx/CoMeT/lib/sniper -c /home/dzx/CoMeT/config/base.cfg --general/total_cores=4 --general/output_dir=/home/dzx/CoMeT/test/test-installation --config=/home/dzx/CoMeT/config/nehalem.cfg --config=/home/dzx/CoMeT/config/gainestown_3D.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/dzx/CoMeT/test/test-installation/sim.scripts.py -g --hooks/script0args= -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmp8Pk2EE/run_benchmarks']

*** Configuration error ***
Configuration value traceinput/benchmarks not found.

[SNIPER] End
[SNIPER] Elapsed time: 0.15 seconds
[RECORD-TRACE] Using the Pin frontend (sift/recorder)
[SIFT_RECORDER] Running /home/dzx/CoMeT/pin_kit/pin -mt -injection child -xyzzy -ifeellucky -follow_execv -t /home/dzx/CoMeT/sift/recorder/obj-intel64/sift_recorder -verbose 1 -debug 0 -roi 0 -roi-mpi 0 -f 0 -d 0 -b 0 -o /tmp/tmp8Pk2EE/run_benchmarks -e 1 -s 0 -r 1 -pa 0 -rtntrace 1 -stop 0    -- /home/dzx/CoMeT/test/test-installation/test
[SIFT_RECORDER:0:0] Output = [/tmp/tmp8Pk2EE/run_benchmarks.app0.th0.sift]
[SIFT_RECORDER:0:0] Response = [/tmp/tmp8Pk2EE/run_benchmarks_response.app0.th0.sift]
[SIFT_RECORDER:0:0] Error: Unable to open the output file /tmp/tmp8Pk2EE/run_benchmarks.app0.th0.sift`

It seems that some temp files cannot be generated properly. Is it related to the virtual machine I am using? I would appreciate it if you can give me some help.

**Comment by shailjap29** [comment]
Please run inside docker for successful simulation, upon which all test cases will pass. Instructions to run inside docker are included in the README file. Let us know if you still face issues.

**Comment by jeremydzx** [comment]
Thank you for your attention. Actually I do run inside docker, and once successfully ran an application like step 2. But after I compiled some benchmarks, both Step 2 and Step 3.4 are failed now with the error outputs I displayed above.
However, I can successfully run the python file `run.py` inside `simulationcontrol`, i.e. running benchmark under the Sniper simulator is ok. So maybe I can ignore this problem just for now.

**Comment by shailjap29** [comment]
*** Configuration error ***
Configuration value traceinput/benchmarks not found.

The above error would come only if you use run-sniper script inside CoMeT/benchmarks folder. Please use run-sniper script inside main CoMeT folder for test/test-installation program. Are you running test-installation program from inside benchmarks folder?

**Comment by jeremydzx** [comment]
No. I'm pretty sure I use the run-sniper inside main CoMeT folder.

![image](https://user-images.githubusercontent.com/84257567/183293177-8da2ab61-cc00-4960-ba4e-344f6c46692f.png)

**Comment by shailjap29** [comment]
The file error_log contains the error you shared in the first post?

**Comment by jeremydzx** [comment]
Both test/test-installation and test/example were failed. And I tried many other ways including trying to run as a root. 

In the end, after all those failed attempts, I clone the codes and follow the instructions to install them again. Surprisingly, no error is reported this time, and everything is running successfully. I think maybe I made some mistakes during the installation before.

So everything is ok now. Sorry for wasting your time, and thanks for all your advice.
